terraform/chef-automate/aws/chef_automate.tf

```diff
@@ -25,6 +25,14 @@ resource "aws_security_group_rule" "ingress_allow_22_tcp_all" {
   security_group_id = "${aws_security_group.chef_automate.id}"
 }
 
+resource "aws_security_group_rule" "ingress_allow_4222_tcp_all" {
```

Comment:
I think this is actually in master now. Probably shows up in the diff because I put it somewhere slightly different in my scrap branch, but this can probably be removed from the commit.